src/cascade/dismod/db/wrapper.py

```diff
@@ -73,8 +73,8 @@ def _validate_data(table_definition, data):
             elif expected_type is str:
                 if len(data) > 0:
                     # Use iloc to get the first entry, even if the index doesn't have 0.
-                    actual_type = type(data[column_name].iloc[0])
-                    correct = np.issubdtype(actual_type, np.str_) or actual_type == type(None)
+                    entry = data[column_name].iloc[0]
+                    correct = np.issubdtype(type(entry), np.str_) or entry is None
```

Comment:
thank you for fixing this! I should not have left that.

Reply:
There's another weird one in there where I just disabled the linter. I couldn't figure out how to do it more cleanly.